doc/man1/openssl-verification-options.pod

```diff
@@ -36,44 +39,47 @@ L<X509_VERIFY_PARAM_set_flags(3)>, and L<X509_check_host(3)>.
 
 =head2 Trust Anchors
 
-In general, according to RFC 4158 and RFC 5280, a I<trust anchor> is
-any public key and related subject distinguished name (DN) that
-for some reason is considered trusted
+In general, according to RFC 4158 and RFC 5280, a I<trust anchor> is just
+a public key (with algorithm and any parameters) and an associated subject name
+that for some reason is considered trusted
 and thus is acceptable as the root of a chain of certificates.
 
-In practice, trust anchors are given in the form of certificates,
-where their essential fields are the public key and the subject DN.
-In addition to the requirements in RFC 5280,
-OpenSSL checks the validity period of such certificates
-and makes use of some further fields.
-In particular, the subject key identifier extension, if present,
-is used for matching trust anchors during chain building.
-
-In the most simple and common case, trust anchors are by default
-all self-signed "root" CA certificates that are placed in the I<trust store>,
+In practice, trust anchors are given in the form of certificates.
+Commonly they are self-signed root CA certificates.
+Their most essential fields are the subjectPublicKeyInfo and
+the subject distinguished name (DN), while the subject key identifier extension
+is generally optional but highly recommended to be included in CA certificates.
+In addition to the requirements given in RFC 5280,
+OpenSSL checks the validity period of all certificates including trust anchors
+and makes use of further fields and extensions
+such as subject key identifier and extended key usage.
+The (regular) key usage extension is ignored for trust anchor certificates.
+
+In the most simple and common case, trust anchors are by default all
+self-signed certificates that have been placed in the I<trust store>,
 which is a collection of certificates that are trusted for certain uses.
 This is akin to what is used in the trust stores of Mozilla Firefox,
 or Apple's and Microsoft's certificate stores, ...
 
-From the OpenSSL perspective, a trust anchor is a certificate
-that should be augmented with an explicit designation for which
+From the OpenSSL perspective, a trust anchor is a certificate that,
+in addition to any included extended key usage (EKU) X.509v3 extension,
+should be augmented locally with an explicit designation for which
 uses of a target certificate the certificate may serve as a trust anchor.
```

Comment:
This is clear as mud. Please rewrite to say something clear and concrete.

Reply:
changed